Taskfile.yml

```diff
@@ -140,6 +141,66 @@ tasks:
       vars:
         COMPONENT: "{{.TASK}}"
 
+  lint-check:
+    deps: ["lint-venv"]
```

Comment:
Is this `deps[]` removable given all the tasks that `link-check` calls has specified the same dependency?

Reply:
Same question with the `deps[]` in task `lint-fix`.

Reply:
Good catch! Fixed.